contentcuration/contentcuration/static/js/bundle_modules/base.js

```diff
@@ -1,62 +1,66 @@
-var attachfastclick = require("fastclick");
+var attachfastclick = require('fastclick');
 
-var $ = require("jquery");
-var get_cookie = require("utils/get_cookie");
+var $ = require('jquery');
+var get_cookie = require('utils/get_cookie');
 global.$ = $;
```

Comment:
It is necessary, because when the code is bundled, it is automatically wrapped in a function scope. The module is not the final code.